fiscalyear.py

```diff
@@ -38,43 +34,52 @@ def fiscal_calendar(start_year=None,
     :type start_month: int or str
     :param start_day: The first day of the first month of the fiscal year
     :type start_day: int or str
-    :raises AssertionError: If ``start_year`` is not ``'previous'`` or ``'same'``
-    :raises TypeError: If ``start_month`` or ``start_day`` is not an int or int-like string
+    :raises TypeError: If ``start_year`` is not a ``str``.
+    :raises ValueError: If ``start_year`` is not ``'previous'`` or ``'same'``
+    :raises ValueError: If ``start_month`` or ``start_day`` is not an int or int-like string
```

Comment:
Can you actually wrap this and the previous line onto a new line? I just noticed they go over 80 characters and flake8/pycodestyle aren't happy. Maybe someday I'll add flake8 testing to Travis.